File: JORGE_HORTA/Trabajos_Java/Practica1/src/main/java/com/poo/sis401s/practica1/Practica1.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.poo.sis401s.practica1;
import java.util.Scanner;
import com.poo.sis401s.OBJETO.Persona;
import java.util.ArrayList;
import com.poo.sis401s.OBJETO.Direccion;
/**
 *
 * @author lobog
 */
public class Practica1 {

    public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      ArrayList<Persona> alumnos = new ArrayList<>();
      for (int i = 0; i < 14; i++){
          Persona persona = new Persona();
          Direccion direccion = new Direccion();
          System.out.println("====ALUMNO===="+ (i + 1)+"====");
          System.out.print("Nombre: ");
          persona.setNombre(sc.nextLine());
          System.out.print("Edad: ");
          persona.setEdad(sc.nextInt());
          sc.nextLine();
          System.out.print("Test: ");
          persona.setTest(sc.nextLine());
          System.out.print("Estatura:");
          persona.setEstatura(sc.nextDouble());
          System.out.print("Peso:");
          persona.setPeso(sc.nextDouble());
          sc.nextLine();
          System.out.println("----DIRECCION----");
          
          System.out.print("Calle: ");
          direccion.setCalle(sc.nextLine());
          System.out.print("Numero: ");
          direccion.setNumero(sc.nextInt());
          System.out.print("Colonia: ");
          direccion.setColonia(sc.nextLine());
          sc.nextLine();
          System.out.print("Codigopostal: ");
          direccion.setCodigopostal(sc.nextInt());
          System.out.print("Estado: ");
          sc.nextLine();
          direccion.setEstado(sc.nextLine());
          System.out.print("Pais: ");
          direccion.setPais(sc.nextLine());
          
          persona.setDireccion(direccion);
          alumnos.add(persona);
          System.out.println();
      }
       System.out.println("====ALUMNOS====");
       for (Persona persona : alumnos){
        System.out.println("Nombre: "+ persona.getNombre());
        System.out.println("Edad: "+ persona.getEdad());
        System.out.println("Test: "+ persona.getTest());
        System.out.println("Estatura: "+ persona.getEstatura());
        System.out.println("Peso: "+ persona.getPeso());
      
        System.out.println("----DIRECCION----");
        
        
        System.out.println("Calle: "+ persona.getDireccion().getCalle());
        
        System.out.println("Numero: "+ persona.getDireccion().getNumero());
        
        System.out.println("Colonia: "+ persona.getDireccion().getColonia());
        
        System.out.println("Codigopostal: "+ persona.getDireccion().getCodigopostal());
        
        System.out.println("Estado: "+ persona.getDireccion().getEstado());
        
        System.out.println("Pais: "+persona.getDireccion().getPais());
      
        System.out.println("--------");
        
          
      }
      }
    }

    

File: JORGE_HORTA/Trabajos_Python/Programacion_objetos/Practica1.py
from Objeto.Persona import Persona
from Objeto.Direccion import Direccion
alumnos = []
for i in range(14):

    persona = Persona()
    direccion = Direccion()

    print("Alumno", i + 1,"====")
    print("Nombre: ") 
    persona.nombre = input()
    print("Edad: ")
    persona.edad = int(input())
    print("Test: ")
    persona.test = input()
    print("Estatura: ")
    persona.estatura = float(input())
    print("Peso: ")
    persona.peso = float(input())

    print("----DIRECCION----")

    print("Calle: ")
    direccion.calle = input()
    print("Numero: ")
    direccion.numero = int(input())
    print("Colonia: ")
    direccion.colonia = input()
    print("Codigopostal: ")
    direccion.codigopostal = int(input())
    print("Estado: ")
    direccion.estado = input()
    print("Pais: ")
    direccion.pais = input()
    persona.direccion = direccion
    alumnos.append(persona)
    print()

    print("====ALUMNOS====")

for persona in alumnos:

    print("Nombre: ",persona.nombre)
    print("Edad: ",persona.edad)
    print("Test: ",persona.test)
    print("Estatura: ",persona.estatura)
    print("Peso: ",persona.peso)
    
    print("====DIRECCION====")
    print("Calle: ",persona.direccion.calle)
    print("Numero: ",persona.direccion.numero)
    print("Colonia: ",persona.direccion.colonia)
    print("Codigopostal: ",persona.direccion.codigopostal)
    print("Estado: ",persona.direccion.estado)
    print("Pais: ",persona.direccion.pais)
    print("==========")
